style.css

```diff
@@ -278,3 +278,83 @@ body {
   width: 100px;
   height: 100px;
 }
+
+#contact-form-section {
+  background: #f9f9fa;
+}
+
+.contact {
+  margin-top: 31.98px;
+  margin-left: 18px;
+  margin-bottom: 20px;
+}
```

Comment:
- [ ] Kindly use `display: flex` to layout the elements of your contact section, it is one of the project requirements and will help you a lot when you create the desktop version.